service/controller/resource/alerting/alertmanagerconfigsecret/test/case-0-cluster-api.golden

```diff
@@ -0,0 +1,446 @@
+global:
+  resolve_timeout: 5m
+  slack_api_url: https://slack
+
+templates:
+- '*.tmpl'
+
+route:
+  group_by: [alertname, cluster_id, installation, status]
+  group_interval: 15m
+  group_wait: 30s
+  repeat_interval: 4h
+  receiver: root
+  routes:
+  - receiver: heartbeat
+    match:
+      severity: heartbeat
+    continue: false
+    group_interval: 1s
+    group_wait: 1s
+    repeat_interval: 15s
+
+    # Team Ops Opsgenie
+  - receiver: opsgenie_router
+    match:
+      severity: page
+    continue: true
+
+  # Project Customer Slack
+  - receiver: sig_customers_slack
+    match_re:
+      severity: notify
+      team: se
+    repeat_interval: 24h
+    continue: false
+
+  # SIG Customer Slack
+  - receiver: sig_customers_slack
+    match_re:
+      severity: page|notify
+      sig: customers
+    repeat_interval: 24h
+    continue: true
+
+  - receiver: team_biscuit_slack
+    match:
+      alertname: ServiceLevelBurnRateTooHigh
+      cluster_type: management_cluster
+    continue: false
+
+  # Service Level slack -- chooses the slack channel based on the provider
+  
+  - receiver: team_firecracker_slack
+  
+    match:
+      alertname: ServiceLevelBurnRateTooHigh
+    continue: false
+
+  # Team Atlas Slack (team)
+  - receiver: team_atlas_slack
+    match_re:
+      severity: page
+      team: atlas
+    continue: false
+
+  # Team Batman Slack (team)
+  - receiver: team_batman_slack
+    match_re:
+      severity: page|notify
+      team: batman
+    continue: false
+
+  # Team Biscuit Slack (team)
+  - receiver: team_biscuit_slack
+    match_re:
+      severity: page
+      team: biscuit
+    continue: false
+
+  # Team Celestial Slack (team)
+  - receiver: team_celestial_slack
+    match_re:
+      severity: page|notify
+      team: celestial
+    continue: false
+
+  # Team Firecracker Slack (team)
+  - receiver: team_firecracker_slack
+    match_re:
+      severity: page|notify
+      team: firecracker
+    continue: false
+
+  # Team Ludacris Slack (team)
+  - receiver: team_ludacris_slack
+    match_re:
+      severity: page|notify
+      team: ludacris
+    continue: false
+
+  # Team Rocket Slack (team)
+  - receiver: team_rocket_slack
+    match_re:
+      severity: page|notify
+      team: rocket
+    continue: false
+
+  # Team Ops Slack
+  - receiver: team_ops_slack
+    match_re:
+      severity: page|notify
+    continue: true
+
+receivers:
+- name: root
+
+- name: heartbeat
+  webhook_configs:
+  - url: https://api.opsgenie.com/v2/heartbeats/test-heartbeat/ping
+    http_config:
+      basic_auth:
+        password: opsgenie-key
+    send_resolved: false
```

Comment:
Those top/installation level heartbeat are gone, heartbeat are now dynamically configure by PMO. We should remove this.

Reply:
You are right